test/e2e/metering_upgrade_test.go

```diff
@@ -18,8 +19,9 @@ const (
 	// TODO: support upgrading from a custom registry image instead
 	// of defaulting to the redhat-operators CatalogSource and waiting
 	// for GA-ed channels to be exposed.
-	upgradeFromCatalogSource          = "redhat-operators"
+	upgradeFromCatalogSource          = "custom-metering-image"
 	upgradeFromCatalogSourceNamespace = "openshift-marketplace"
+	upgradeFromImage                  = "quay.io/tflannag/metering-index:4.5"
```

Comment:
This is an index image containing a single bundle image where that bundle image contains a 4.5 metering CSV with an overridden metering-ansible-operator image (which contains the many-to-many 4.5 fixes).